pkgs/development/tools/build-managers/bazel/bazel_3/default.nix

```diff
@@ -169,8 +171,6 @@ stdenv.mkDerivation rec {
   sourceRoot = ".";
 
   patches = [
-    ./python-shebang.patch
```

Comment:
Why was this removed?

Reply:
Not needed anymore, since integrated upstream.

Reply:
Forgot to remove the patch file though...

Reply:
Removed the patch file and added a note + link in the commit message.